README.md

```diff
@@ -50,11 +66,7 @@ const text = cgt.gen({
 fs.writeFileSync(`subgraphQuery.ts`, text);
 ```
 
-## Supported Methods
-
-Currently supported generation methods are `plain` (raw async/await) and `reactquery` (react hooks which query subgraph--suitable for direct usage in frontend code)
```

Comment:
how do we explain these different formats without having this secton here?

Reply:
I imagined line 8 covered it, but making some changes to make it more explicit.